sdk/resourcemanager/scvmm/armscvmm/constants.go

```diff
@@ -10,9 +10,23 @@ package armscvmm
 
 const (
 	moduleName    = "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/scvmm/armscvmm"
```

Comment:
Seems this module name doesn't align to the namespace 
https://github.com/Azure/azure-sdk/issues/7013#issuecomment-1849354704